etc/webpack/utils/utils.js

```diff
@@ -1,17 +1,15 @@
-
 function getRuleJS(...includePaths) {
     return {
         test: /\.js$/,
         include: includePaths,
         use: [
             {
-                loader: 'babel-loader',
+                loader: 'babel-loader'
```

Comment:
Might be good idea to add `eslint-loader` here, so during the build process or dev mode, we can catch errors easily

Reply:
added for prod but not for dev